src/js/components/invoice/ConfirmInvoicePage.jsx

```diff
@@ -226,6 +229,34 @@ class ConfirmInvoicePage extends Component {
       .catch(() => this.props.hideSpinner());
   }
 
+  fetchPrepaymentItems() {
+    const url = `/openboxes/api/invoices/${this.state.values.id}/prepaymentItems`;
+    apiClient.get(url)
+      .then((response) => {
+        const { data, totalCount } = response.data;
+        const lineItemsData = _.map(
+          data,
+          val => ({
+            ...val,
+            totalAmount: val.totalAmount * (-1),
```

Comment:
@DominikaHatala I think it would be better to have these prapayment items with value `* -1` available from the invoice domain level because this probably will be included in the invoice view page too.
cc @jmiranda

Reply:
Another thing I just realized. What if the line is a `DISCOUNT` order adjustment with an already negative value?

Reply:
Agreed on the first comment. I think the second comment requires Kelsey to respond.